googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
The full block should be within `if debug`. I don't want to initialize this even if it's not required.